perceval/backends/core/gerrit.py

```diff
@@ -277,22 +280,31 @@ class GerritClient():
     :param repository: URL of the Gerrit server
     :param user: SSH user to be used to connect to gerrit server
     :param max_reviews: max number of reviews per query
+    :param blacklist_reviews: exclude the reviews of this list while fetching
+    :param disable_host_key_check: disable host key controls
+    :param port: SSH port
+    :param archive: collect issues already retrieved from an archive
+    :param from_archive: it tells whether to write/read the archive
     """
     VERSION_REGEX = re.compile(r'gerrit version (\d+)\.(\d+).*')
     CMD_GERRIT = 'gerrit'
     CMD_VERSION = 'version'
     MAX_RETRIES = 3  # max number of retries when a command fails
     RETRY_WAIT = 60  # number of seconds when retrying a ssh command
 
-    def __init__(self, repository, user, max_reviews, blacklist_reviews=[],
-                 disable_host_key_check=False, port=PORT):
+    def __init__(self, repository, user=None, max_reviews=MAX_REVIEWS, blacklist_reviews=[],
```

Comment:
The parameter `blacklist_reviews` cannot be initialized to an empty list. This might cause side-effect errors. Initialize it to `None` and inside the method, set it to an empty list when is `None`.